services/skus/credentials.go

```diff
@@ -332,14 +295,81 @@ func (s *Service) CreateOrderItemCredentials(ctx context.Context, orderID, itemI
 		},
 	}
 
-	err = s.Datastore.InsertSigningOrderRequestOutbox(ctx, requestID, order.ID, orderItem.ID, signingOrderRequest)
-	if err != nil {
+	if err := s.Datastore.InsertSigningOrderRequestOutbox(ctx, requestID, order.ID, orderItem.ID, signReq); err != nil {
 		return fmt.Errorf("error inserting signing order request outbox orderID %s: %w", order.ID, err)
 	}
 
 	return nil
 }
 
+func (s *Service) doCredentialsExist(ctx context.Context, item *model.OrderItem, blindedCreds []string) error {
+	switch item.CredentialType {
+	case timeLimitedV2:
+		return s.doTlv2Exist(ctx, item, blindedCreds)
```

Comment:
question: should Tlv2 be capitalised as its an acronym?

Reply:
Yes, good call. Updated.